.github/workflows/pr-build.yml

```diff
@@ -3,43 +3,77 @@ on:
   push:
     branches:
       - main
-  pull_request_target:
-    branches:
-      - main
+  pull_request:
 
 jobs:
+  extract-args-java-11:
+    uses: ./.github/workflows/extract-args.yml
+    with:
+      file: java-11/args.env
+      image: hypertrace/java-build
+
+  extract-args-java-21:
+    uses: ./.github/workflows/extract-args.yml
+    with:
+      file: java-21/args.env
+      image: hypertrace/java-build
+
   build:
     runs-on: ubuntu-22.04
+    strategy:
+      matrix:
```

Comment:
So the matrix is just to support the scan right? If we leave the publish as a standard multiplatform build do we get the manifest too?

Reply:
I believe that will be the case based on what I saw from example of build publish action. Will verify after cutting release

Reply:
Got it. Sorry, left this comment before looking at publish and meant to remove it when I saw we went back to the other approach there.